### DISCUSSION TITLE
Data set beyond a single state

### DISCUSSION THREAD

**Discussion by niranjan-sathindran** [synthetichealth/synthea#1267] [Q&A]
Is there a reason why default is limited to a single state (MA)? Would it be possible to generate a data set that'll randomly generate data across the entire country?

**Comment by jawalonoski** [comment]
Historically, Synthea was used to create "SyntheticMass" -- a million synthetic patients in the state of Massachusetts. That is why Massachusetts remains the default state.

You can specify a different state and/or city on the command line (see [Basic Setup and Running](https://github.com/synthetichealth/synthea/wiki/Basic-Setup-and-Running)). You can also replace the demographics (see [Other Areas](https://github.com/synthetichealth/synthea/wiki/Other-Areas), [Demographics for Other Areas](https://github.com/synthetichealth/synthea/wiki/Demographics-for-Other-Areas), and related wiki articles).

If you want to run multiple states, we recommend that users write a script that simply executes Synthea multiple times for each state desired. We do this ourselves when we want to generate a population for the entire United States.

**Comment by idhamari** [comment]
I think this would be a good feature to add, adding multiple states or special words example  -p "state1, state2" or "All" for all states.
